<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>561</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <widget class="QWidget" name="saleManagePage">
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>251</width>
        <height>321</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>249</width>
         <height>247</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Sale Cars</string>
       </attribute>
       <widget class="QLabel" name="Brand">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>68</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Brand</string>
        </property>
       </widget>
       <widget class="QComboBox" name="sellFactoryComboBox">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="sellModelComboBox">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="Model">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>68</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>68</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="sellPriceLineEdit">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>68</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Amount</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sellAmountSpinBox">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>45</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="sellLastLabel">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>130</y>
          <width>91</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Remain:000</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>68</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="sellTotalLineEdit">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>170</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="sellOkBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>91</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sellCancelBtn">
        <property name="geometry">
         <rect>
          <x>141</x>
          <y>210</y>
          <width>91</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>249</width>
         <height>247</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Cars Recording</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QListWidget" name="dailyList">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>256</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
